doc/examples.md

```diff
@@ -0,0 +1,108 @@
+# Examples: Getting Started using Padertorch
+
+Padertorch's main purpose is to provide an open interface that simplifies tedious work and works with as many different 
+workflows as possible. Due to the open design of Padertorch, there is no "right" or "wrong" way to train a neural network.
+However, to make getting started easier, we have provided several examples with our recommended workflow when using Padertorch. 
+
+We recommend the usage of the following components when using Padertorch:
+  - [sacred](): Experiment Organization / Ensuring reproducability
+  - [lazy_dataset](https://github.com/fgnt/lazy_dataset): Data preparation and on-the-fly data generation
+  - [Padertorch Configurable](): Adjustment of training hyperparameters
+  
+The examples mostly use all of these components, whereas the simple examples  (See [Getting Started]()) omit some of these components to reduce the overall complexity of the example as much as possible.
+
+
+## Basic Structure of a Training:
+
+Each example several has several steps:
+  - Experiment setup (directory creation, reproducability, logging)
+  - Data preprocessing
+  - Training
+  - (Optional:) Evaluation
+
+### Experiment setup
+During the experiment setup, the experiment directory is created. We recommend using [Sacred]() to monitor each training.
+Inside this directory, the checkpoints of the training as well as the event files for visualization are saved. 
+The examples using Padertorch Configurable and Sacred also save the model hyperparameters in this folder and log the 
+changes over multiple starts of the same training.
+
+In its most basic form this step looks as simple as:
+``` python
+import os
+
+storage_dir = pt.io.get_new_storage_dir('my_experiment')
+```
+For more information regarding the usage of Configurable or Sacred 
+see [Configurable Introduction]() and [Sacred Introduction]()
+
+### Data preprocessing
+Data preprocessing describes the creation of an iterable object (e.g. a list of examples)for the Padertorch Trainer.
+In our examples, we use [lazy_dataset](https://github.com/fgnt/lazy_dataset) to obtain an iterable object and map all necessary transformations onto this iterable.
+
+``` python
+import lazy_dataset.database
+
+def mapping_function(example):
+    example['new_keys'] = some_transform_function(example)
+    return example
+
+iterable = lazy_dataset.database.DictDatabase(data)
```

Comment:
Is it possible to add a link to a property of a class in github?
If yes, we should point the user to the data property in: https://github.com/fgnt/lazy_dataset/blob/master/lazy_dataset/database.py

Reply:
Unfortunately, it's not possible to add a hyperlink in a code block. I could link to the lazy_dataset data property before the example code, instead. Do you think that would help?

Reply:
Good idea.